p5js/YOLO/sketch.js

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2018 ml5
+// 
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+/* ===
+ML5 Example
+YOLO
+Real time Object Detection
+=== */
+
+let video;
+let yolo;
+let objects = [];
+
+function setup() {
+  createCanvas(640, 410);
+  video = createCapture(VIDEO, onVideoLoaded);
+  video.hide();
+  select('#start').mousePressed(function() {
+    detect();
+  });
+}
+
+function draw() {
+  image(video, 0, 0, 640, 410);
+  objects.forEach(function(object) {
```

Comment:
`forEach` loops aren't commonly used in p5 examples so I might just use a regular for loop or a `for..of` loop?